pkg/grpc/actions/organizations/list_organizations_test.go

```diff
@@ -0,0 +1,41 @@
+package organizations
+
+import (
+	"context"
+	"testing"
+
+	uuid "github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/superplanehq/superplane/pkg/authentication"
+	"github.com/superplanehq/superplane/pkg/authorization"
+	"github.com/superplanehq/superplane/pkg/database"
+	"github.com/superplanehq/superplane/pkg/grpc/actions/auth"
+	"github.com/superplanehq/superplane/pkg/models"
+	protos "github.com/superplanehq/superplane/pkg/protos/organizations"
+)
+
+func Test__ListOrganizations(t *testing.T) {
```

Comment:
We should have tests to check that an organization that is not accessible to a user is not returned in the response when listing organizations.